Src/BootCamp.Chapter/Lesson4.cs

```diff
@@ -0,0 +1,126 @@
+﻿using System;
+
+namespace BootCamp.Chapter
+{
+    class Lesson4
+    {
+
+        public static void Demo()
+        {
+            bool backtotop = true;
```

Comment:
Back to top is how while loop works, but instead, it should refer to a logical condition

Reply:
Can you explain this for me please? im not sure I understand what you mean.